artifacts

```diff
@@ -24,7 +24,11 @@ setup() {
 }
 
 prolong() {
-    echo "TODO"
+    echo ${NAME} | grep -qE '^[^/]+\:staging\-[0-9]{10}\.[0-9a-f]+\.[^./]+\.[0-9]+$'
+    ARTIFACT_TARGET=$(echo ${NAME} | sed -E 's/(^[^/]+\:)staging(\-[0-9]{10}\.[0-9a-f]+\.[^./]+\.[0-9]+)$/\1prolonged\2/')
+    echo "copying '${NAME} to '${ARTIFACT_TARGET}'"
+
```

Comment:
Could you add the following outputs:
* name: with the value of the prolonged artifacts name
* link: with the full link with the prolonged artifacts name